nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/GetHealthCheckConfiguration.java

```diff
@@ -109,6 +111,7 @@
     private final String machineMemoryUsagePropertyName = "machineMemoryUsage";
     private final String hoggingThreadsPropertyName = "hoggingThreads";
     private final String stuckThreadsPropertyName = "stuckThreads";
+    private final String mpHealthcheckPropertyName = "mpHealthcheck";
```

Comment:
Could probably just be _mpHealth_ - it's more accurate (the spec is actually just "MicroProfile Health")